src/components/PDiskPopup/PDiskPopup.tsx

```diff
@@ -50,6 +57,18 @@ export const preparePDiskData = (data: PreparedPDisk, nodeHost?: string) => {
         pdiskData.push({label: 'Device', value: Device});
     }
 
+    if (withDeveloperUILink && valueIsDefined(NodeId) && valueIsDefined(PDiskId)) {
+        const pDiskInternalViewerPath = createPDiskDeveloperUILink({
+            nodeId: NodeId,
+            pDiskId: PDiskId,
+        });
+
+        pdiskData.push({
+            label: 'Links',
```

Comment:
There is no `i18n` for this files, I don't want mix this chores with this feature (it creates big diffs)